docs/content/preview/faq/yugabytedb-managed-faq.md

```diff
@@ -92,7 +92,7 @@ During a [maintenance window](../../yugabyte-cloud/cloud-clusters/cloud-maintena
 
 #### Cluster (YugabyteDB) version upgrade
 
-To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte upgrades your cluster database to the [latest version](#what-version-of-yugabytedb-does-my-cluster-run-on).
+To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte upgrades your cluster database to the [latest version](#what-version-of-yugabytedb-does-my-cluster-run-on). A backup is automatically run before a database upgrade.
```

Comment:
I would not add this.  The implication here is that the backup is immediately before the upgrade, but the reality is we run it a few hours before, as backups on large clusters take multiple hours.